config.go

```diff
@@ -288,8 +288,8 @@ func continueTask(s string, cont bool) bool {
 
 	fmt.Print(bold(green(arrow)+" "+s), bold(postFix))
 
-	len, err := fmt.Scanln(&response)
-	if err != nil || len == 0 {
+	i, err := fmt.Scanln(&response)
```

Comment:
Maybe `len` was not the best descriptor for the return, but I think `i` is not much better. Maybe the `n` which is the same as the function header `func Scanln(a ...interface{}) (n int, err error)` would be a better choice.

Reply:
>It returns the number of items successfully scanned. If that is less than the number of arguments, err will report why.

Maybe we don't even need the variable.

Reply:
True, maybe it can even be removed (if it's like that it would never evaluate it anyway)

Reply:
You are right, judging by the function description, the second comparison will not happen. Variable removed in 0347d1d3d69671a7f482193a1b5e25ef5bd0afc8